CHANGELOG.md

```diff
@@ -36,7 +36,8 @@ release.
 ## [Unreleased]
 
 ### Added
-- Adds support for LO [#11](ihttps://github.com/DOI-USGS/SpiceQL/issues/11)
+- Adds support for LO [#11](https://github.com/DOI-USGS/SpiceQL/issues/11)
+- Adds support for Smart1 [#16](https://github.com/DOI-USGS/SpiceQL/issues/16)
```

Comment:
Seems the changelog from another PR was included?

Reply:
There was a typo in that line -- it was ihttps:// and this corrects the link

Reply:
ah I see now, in the future it should probably be in a separate PR. But we havn't adopted that policy here yet.